### PR TITLE
add integr8ly (heimdall-operator image) to image-mirroring

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**JIRA**
https://issues.redhat.com/browse/INTLY-4819

Adding config for integr8ly/heimdall-operator to have it's image mirrored to it's quay.io repository.
There's still some stuff to do here with secrets/credentials. Chatting with DPTP team about what credentials need to be shared and mirrored here but pushing for an early review.

I got a PGP key from the team but not sure where I should put that.

### CODE REVIEW

File: core-services/image-mirroring/integr8ly/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+approvers:
+- maleck13
+- mikenairn
+options: {}
```
Comment (mikenairn) [inline core-services/image-mirroring/integr8ly/OWNERS]: @laurafitzgerald Can you remove Craig and add yourself, Paul and Steven who are all members of the openshift org.

Comment (laurafitzgerald) [reply core-services/image-mirroring/integr8ly/OWNERS]: @mikenairn I see Craig listed in other owner files. Should we remove him from those?

**Pull request merged**